### DISCUSSION TITLE
When you close History the sidebar is closed.

### DISCUSSION THREAD

**Discussion by renfrow** [piroor/treestyletab#3185] [Q&A]
Is there a setting to prevent this:
When you open History (Ctrl-H), and then close it, the sidebar is closed.

I would like TST to re-open automatically and not have to click up on the TST button.

**Comment by irvinm** [comment]
I don't think you can get around that behavior as that is how Firefox handles Ctrl-H followed by another Ctrl-H.  However, you could do Ctrl-H to bring up history and then "F1" to bring up TST.

**Reply by piroor** [reply]
He is right. TST is just a sidebar panel and there is no way to go back to TST's sidebar panel after other panels are closed. It is the only way: switching to TST directly from History sidebar.

**Reply by ocfox** [reply]
Thank you, save my life.
